pallets/funding/src/functions/misc.rs

```diff
@@ -248,9 +248,6 @@ impl<T: Config> Pallet<T> {
 		let project_details = ProjectsDetails::<T>::get(project_id).ok_or(Error::<T>::ProjectDetailsNotFound)?;
 		let total_fee_allocation = Self::calculate_fee_allocation(project_id)?;
 
-		// Calculate the percentage of target funding based on available documentation.
-		// A.K.A variable "Y" in the documentation. We mean it to saturate to 1 even if the ratio is above 1 when funding raised
-		// is above the target.
 		let percentage_of_target_funding = Perquintill::from_rational(
 			project_details.funding_amount_reached_usd,
 			project_details.fundraising_target_usd,
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +257]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Here we still have the Y calculation for rewards info. Should be removed as well. <!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/Polimec/polimec-node/363?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->